src/app/components/charts/chart.service.js

```diff
@@ -239,10 +241,138 @@ angular.module('transmartBaseUi')
 
     chartService.renderAll = function (charts) {
       angular.forEach (charts, function (chart) {
-         // console.log(chart);
         chart.render();
       });
     };
 
+    /*******************************************************************************************************************
+     * Cohort chart service
+     */
+    var COHORT_SUBJECTS = [], CROSSFILTER = null, DIMENSIONS = {}, GROUPS = {}, CHARTS = [], CHART_IDS = 0,
+      NEW_LABELS = [], NEW_TYPES = [], NEW_NAMES = [];
```

Comment:
I'm a little bit concern on this part. If we declare everything global is hard to trace and prone to scoping error. Shall we define variables as it's used in the associated function? Also could you maintain the camelcase style for the variable naming?